cybench/runs/agml_workshop.py

```diff
@@ -17,20 +18,34 @@
     KEY_LOC,
     KEY_YEAR,
     KEY_TARGET,
+    KEY_CROP_SEASON,
+    MIN_INPUT_YEAR,
+    MAX_INPUT_YEAR,
     SOIL_PROPERTIES,
     METEO_INDICATORS,
     RS_FPAR,
     STATIC_PREDICTORS,
     TIME_SERIES_PREDICTORS,
-    CROP_CALENDAR_ENTRIES,
+    CROP_CALENDAR_DOYS,
+    CROP_CALENDAR_DATES,
 )
 
 
 class LSTMModel(BaseModel, nn.Module):
     def __init__(
-        self, num_rnn_layers=1, rnn_hidden_size=64, num_outputs=1, *args, **kwargs
+        self,
+        time_series_have_same_length=False,
```

Comment:
doesnt the model assume this is the case? In my understanding it doesnt support an alternative. Doesnt it make this code redundant?

Reply:
We run workshop results validation with data that does not need aggregation.